app/src/main/java/com/peacecorps/pcsa/LoginActivity.java

```diff
@@ -0,0 +1,81 @@
+package com.peacecorps.pcsa;
+
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.preference.PreferenceManager;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Spinner;
+import android.widget.Toast;
+
+public class LoginActivity extends AppCompatActivity  implements AdapterView.OnItemSelectedListener {
+
+    private Toolbar toolbar;
+    private Button loginButton;
+    private EditText name;
+    private Spinner country;
+    private String selected_country;
+    SharedPreferences.Editor editor;
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login);
+        toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+
+        editor = PreferenceManager.getDefaultSharedPreferences(LoginActivity.this).edit();
```

Comment:
Sorry my comment is on the wrong line. you have set the first time as a shared preference right? I am asking whether is this the same variable which is checked in the tutorial screens. (when the app is first run, there will be a sort of how to use circle of trust tutorial/instructions screens). I am asking whether are you using the same shared preference which is used to detect the first run there?